### PR TITLE
Fix knowbase browse view in some locales

### PR DESCRIPTION
**Pull request by cconard96**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #13799

### CODE REVIEW

**Review by trasher — APPROVED** [review]

**Review by cedric-anne — CHANGES_REQUESTED** [review]
Same issue exists on `Glpi\Features\TreeBrowse` class.

File: src/Features/TreeBrowse.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public static function showBrowseView(string $itemtype, array $params, $update =
         global $CFG_GLPI;
 
         $ajax_url    = $CFG_GLPI["root_doc"] . "/ajax/treebrowse.php";
-        $loading_txt = addslashes(__('Loading...'));
+        $loading_txt = addslashes(__s('Loading...'));
```
Comment (cedric-anne) [inline src/Features/TreeBrowse.php:64]: `addslashes` is not needed anymore.
```suggestion
        $loading_txt = __s('Loading...');
```

File: src/Html.php [hunk 2]
```diff
@@ -848,7 +848,7 @@ public static function displayDebugInfos($with_session = true, $ajax = false, $r
             }
             foreach ($plugin_tabs as $tab_id => $tab_info) {
                 if (isset($tab_info['display_callable']) && !is_callable($tab_info['display_callable'])) {
-                    trigger_error(sprintf('Debug tab "%s"(%s) display callable is invalid.', $tab_info['title'] ?? '', $tab_id), E_USER_WARNING);
+                    //trigger_error(sprintf('Debug tab "%s"(%s) display callable is invalid.', $tab_info['title'] ?? '', $tab_id), E_USER_WARNING);
```
Comment (cedric-anne) [inline src/Html.php:851]:
```suggestion
                    trigger_error(sprintf('Debug tab "%s"(%s) display callable is invalid.', $tab_info['title'] ?? '', $tab_id), E_USER_WARNING);
```

File: src/Knowbase.php [hunk 3]
```diff
@@ -150,7 +150,7 @@ public static function showBrowseView()
 
         $rand        = mt_rand();
         $ajax_url    = $CFG_GLPI["root_doc"] . "/ajax/knowbase.php";
-        $loading_txt = addslashes(__('Loading...'));
+        $loading_txt = addslashes(__s('Loading...'));
```
Comment (cedric-anne) [inline src/Knowbase.php:153]:
```suggestion
        $loading_txt = __s('Loading...');
```

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**